### DISCUSSION TITLE
Administrator not running

### DISCUSSION THREAD

**Discussion by Ciscodog** [ppy/osu#19653] [Q&A]
### Type

Other

### Bug description

After booting up the game normally, the game believes I am using Administrator, despite the fact I am not. I used a shortcut, so I deleted it and tried booting up the game directly, making sure not to use Admin. This seemed to fail as well. I re-create the shortcut, and I still am left with the message I am in Administrator mode. While this bug has no impeding factors with my game, it is nonetheless an issue I wish to be resolved.

The message that it leaves me reads as follows:
"Running osu! as administrator not improve performance, may break integrations and poses a security risk. Please run the game as a normal user."

To which I mentally respond: I am running it as a normal user!

### Version

2022.731.1

### Logs

[database.log](https://github.com/ppy/osu/files/9286418/database.log)
[network.log](https://github.com/ppy/osu/files/9286419/network.log)
[performance.log](https://github.com/ppy/osu/files/9286420/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9286421/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9286422/updater.log)

**Comment by peppy** [comment]
Have you customised your windows install at all? Are you running any compatibility mode settings on the game?

**Comment by Draylynn** [comment]
I have the same problem after I recently switched to Osu! Lazer. It isn't in compatibility mode, isn't set 'run as admin' (in either system properties permissions). Yet I keep getting the admin warning popup.
The only user account on my Win11 system is the administrator of my PC(?), as it has been for over 20 years through the various OS's

Whilst I understand how some PC users might appreciate this warning, I honestly don't care whether the game is in admin mode or not and don't need constantly reminding about something that cannot be seemingly turned off. Could we get an option to disable this warning please?

**Reply by Joehuu** [reply]
> The only user account on my Win11 system is the administrator of my PC(?), as it has been for over 20 years through the various OS's

As said in https://www.howtogeek.com/709060/what-does-run-as-administrator-mean-in-windows-10/:

> But even if you are using an administrator account on Windows, not every application needs full administrator permissions. In fact, that's bad for security---your web browser shouldn't have full access to your entire operating system. [User Account Control (UAC)](https://www.howtogeek.com/124754/htg-explains-why-you-shouldnt-disable-uac/) limits the permissions that applications have, even when you launch them from an administrator account.

Also see when/why it was initially implemented in the first place: https://github.com/ppy/osu/issues/12520

**Reply by Draylynn** [reply]
I completely understand. That may be an issue for other users, I did state that... I understand why it's implemented, you know, for the people who don't firewall up, download cheats and files blindly, never check anything they install, yada yada. Not here to argue the semantics of internet safety and the increasingly strangled usage of Microslaps operating systems.

> Also see when/why it was initially implemented in the first place: https://github.com/ppy/osu/issues/12520
Not sure what I'm looking at here. Huzzah, someone asked for it?

Asking for a toggle, because it's annoying to be constantly warned about something that cannot be changed without having to change users etc, every. single. time. I fire up OSU! Lazer. Congrats, the warning worked, we know, can we now confirm the warning and toggle it off?
If it cannot be done, I will go back to the old OSU client, who never QQ'd about me being the administrator of my own system.
